front/app/utils/actionTakingRules.ts

```diff
@@ -148,21 +148,14 @@ export const getIdeaPostingRules = ({
       };
     }
 
-    const supportsNativeSurvey =
-      phase?.attributes.participation_method === 'native_survey' ||
-      phase?.attributes.participation_method === 'community_monitor_survey';
-
     // timeline
     if (phase) {
       // not an enabled ideation or native survey or proposals phase
       if (
-        !(
-          (phase.attributes.participation_method === 'ideation' ||
-            phase.attributes.participation_method === 'proposals' ||
-            supportsNativeSurvey) &&
-          phase.attributes.submission_enabled &&
-          disabled_reason !== 'posting_not_supported'
-        )
+        (phase.attributes.participation_method === 'ideation' ||
```

Comment:
This was actually a bug - we shouldn't have been checking for Survey-based phases here. It's handled correctly further down in the component already.

When I fixed this, the re-enabled E2E test passed.